api.planx.uk/lib/hasura/metadata/index.ts

```diff
@@ -11,6 +11,7 @@ interface ScheduledEvent {
 
 export interface CombinedResponse {
```

Comment:
Edge case - we might also need to think about updating logic for sending in Invite to Pay routes  - see `createPaymentSendEvents()` in `api.planx.uk/modules/pay/service/inviteToPay/createPaymentSendEvents.ts`

Ideally this would have been an exhaustive `switch` on `Destination` and this change would have flagged there - I'll make a note to tidy this up and cover this case when adding destinations once this PR has progressed and merged.

Currently an ITP submission would just go to the old BOPS endpoint. This very likely is not a problem realistically speaking.

If BOPS take a while to turn off the old v1 endpoint though we might encounter an ITP submission - I'd suggest we cover this eventuality there as it should be a very quick addition to cover this extra destination.